### PR TITLE
[#372] Multi-country/transnational list filter

### PR DESCRIPTION
**Pull request by wayangalihpratama**
- [x] Implement transnational list [design](https://xd.adobe.com/view/06fdb38e-2dae-4870-a030-e15c185e68e0-53ae/screen/41ba7784-d8e3-481d-aa38-4909d4a32b3a) with static data. Feedback [source](https://docs.google.com/spreadsheets/d/10y4U-AYdkCflxN9rWAB4fuEblDP71eAhQwR-kFvkYhI/edit?pli=1#gid=966469174&range=39:39)
- [x] Load transnational list from BE to multi country filter dropdown
- [x] Show summary count of multi country selected
- [x] Change map color for countries included in multi country selected
- [x] Show country list popover of multi country selected

### CODE REVIEW

File: frontend/src/modules/browse/map-landing.jsx [hunk 1]
```diff
@@ -21,38 +28,64 @@ const MapLanding = ({
...
+    !isEmpty(transnationalOptions);
 
   const clickCountry = (name) => {
     setToggleButton("list");
     updateQuery("country", name);
     history.push(`/browse?country=${name}`);
   };
 
+  const handleChangeTab = (key) => {
+    key === "multi-country" && setCountry(null);
```
Comment (tangrammer) [inline frontend/src/modules/browse/map-landing.jsx:64]: should these lines better expressed as a conditional?

Comment (tangrammer) [reply frontend/src/modules/browse/map-landing.jsx:64]: I mean simple conditional
```js
if(){
} else if(){
}
```

**Review by tangrammer — APPROVED** [review]
LGTM!, just a minor comment, feel free to adapt the code accordingly or just merge as it is

File: frontend/src/modules/browse/map-landing.jsx (hunk 1 above)
Comment (wayangalihpratama) [reply frontend/src/modules/browse/map-landing.jsx:64]: Hmm, yep,
I will change it to a  simple conditional

**Pull request merged**